app/lua53/lutf8lib.c

```diff
@@ -258,6 +258,7 @@ LROT_END(utf8, LROT_TABLEREF(utf8_meta), 0)
 
 
 LUAMOD_API int luaopen_utf8 (lua_State *L) {
+  (void)L;
```

Comment:
There's actually an `UNUSED(v)` macro in `llimits.h`

Reply:
Ah, I hadn't noticed that. Happy to swap over to that both here and on the esp32 branch then. Stay tuned... :)